addon/services/fb.js

```diff
@@ -49,27 +49,9 @@ export default Ember.Service.extend({
     this.setAccessToken(token);
   },
 
-  api(path) {
-    var method = 'GET';
-    var parameters = {};
-    var arg;
-
+  api(path, method = 'GET', parameters = {}) {
```

Comment:
Default parameters? I didn't even know they exist in JS: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters